app/celery/service_callback_tasks.py

```diff
@@ -121,6 +127,8 @@ def create_delivery_status_callback_data(notification, service_callback_api):
         "notification_type": notification.notification_type,
         "service_callback_api_url": service_callback_api.url,
         "service_callback_api_bearer_token": service_callback_api.bearer_token,
+        "template_id": str(notification.template_id),
+        "template_version": notification.template_version,
```

Comment:
I've kept template version as an integer. All the other fields are strings, so it could be changed to `"1"` for consistency, but that might be unexpected.

Reply:
Seems correct – the `GET /template` endpoint expects version to be an integer according to https://docs.notifications.service.gov.uk/python.html#get-a-template-by-id-and-version